Makefile

```diff
@@ -760,14 +760,18 @@ AUTOPEP8_OPTS     += --in-place --aggressive --max-line-length 120
 
 format.ch := $(filter-out include/ceedf.h $(wildcard tests/t*-f.h), $(shell git ls-files '*.[ch]pp' '*.[ch]'))
 format.py := $(filter-out tests/junit-xml/junit_xml/__init__.py, $(shell git ls-files '*.py'))
+format.ot := $(filter-out rust/libceed-sys/c-src/% doc/sphinx/source/CODE_OF_CONDUCT.md doc/sphinx/source/CONTRIBUTING.md, $(shell git ls-files '*.[ch]pp' '*.[ch]' '*.py' '*.jl' '*.md' '*.f90'))
 
 format-c  :
 	$(CLANG_FORMAT) $(CLANG_FORMAT_OPTS) $(format.ch)
 
 format-py :
 	$(AUTOPEP8) $(AUTOPEP8_OPTS) $(format.py)
 
-format    : format-c format-py
+format-ot:
```

Comment:
How about this for a whitespace trimming rule? Do we want to change the list of targeted files in any way?

Reply:
I think this can race with `format-c`. Maybe stripping whitespace for C sources can go into `format-c`.

Reply:
I think only comment blocks are at risk of trailing whitespace right now in C files.

For C, the funkiness comes  from
```
ReflowComments:               false
```
I turned this on to prevent this sort of silliness
```C
/**
  A long comment with one more word than Clang
likes.
**/
```
where we have a line with just one word on it for no particular reason. For some reason that option means that trailing spaces are allowed in comment blocks?

Reply:
Trimmed to .f90 and .md, which aren't covered by other linting